app/jobs/regular/code_review_sync_commits.rb

```diff
@@ -13,6 +13,19 @@ def execute(args)
       github_commit_querier = DiscourseCodeReview.github_commit_querier
 
       repo = DiscourseCodeReview::GithubRepo.new(repo_name, client, github_commit_querier, repo_id: repo_id)
+
+      if args[:skip_if_updated]
```

Comment:
Shouldn't the name be `skip_if_up_to_date`? We definitely don't want to skip if there's something to update.

But the more I think about this, the less I think we need an `if`. We should probably do that check every time. This job is either launched whenever we receive a `push` webhook or every hour. Due to the nature of webhooks, there might be cases where we receive webhooks and there's nothing to update because we've already done the work.

What do you think?

Reply:
I am sorry, but I do not understand. What do you mean we can receive webhooks and there to be nothing to update?

I added this as an optional check because I do not want to make a request to GitHub API everytime we receive a webhook because then it is clear there is something to update.